modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMDashboards.testcase

```diff
@@ -95,6 +95,82 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-179357. Verify that the value of Approved Claims is displayed on MDF Dashboard"
+	@priority = 4
+	test ClaimApprovedValueAppearInDashboard {
+		var budgetValue = 2000;
+
+		task ("Given I get the current amount of requested MDF Claim Value") {
+			var initialValue = PRMDashboards.getLegendValue(
+				chartName = "Request",
+				legendName = "Claim Approved");
+		}
+
+		task ("When I create a new MDF Request") {
+			JSONPRM.createMDFRequest(
+				accountEntryName = "Company Name 1",
+				budgetValueList = ${budgetValue},
+				campaignName = "Campaign Name 1");
+		}
+
+		task ("And I go back to MDF listing page") {
+			PRMNavigator.openSitePage(pageName = "MDF Requests");
+		}
+
+		task ("And I go approve the MDF") {
+			PRMMDFRequest.goToMDF(campaignName = "Campaign Name 1");
+
+			PRMMDFRequest.changeMDFStatus(statusButton = "Approve");
+		}
```

Comment:
Hi @marinaasant0s, here you could use the macro to approve the MDF via API: `JSONPRM.changeMDFStatus`